templates/web/base/admin/extra-metadata-item.html

```diff
@@ -27,7 +27,7 @@
 
         <div class="form-group">
             <label for="metadata-[% i %]-code">[% loc('Code') %]</label>
-            <span class="form-hint" id="metadata-[% i %]-code-hint">[% loc('The code used to store this field value in the database.') %]</span>
+            <span class="form-hint" id="metadata-[% i %]-code-hint">[% loc('The code used to store this field value in the database. Must not contain spaces.') %]</span>
```

Comment:
This is fine; I wonder if we might as well have it as `[% loc('The code used to store this field value in the database.') %] [% loc('Must not contain spaces.') %]</span>` - so that the current translations aren't disturbed by the string changing, and so the smaller string could potentially be reused elsewhere in future without needing new translation. Both very minor and theoretical, but wouldn't hurt.